src/controller/phenomenon.controller.js

```diff
@@ -0,0 +1,12 @@
+const {Phenomenon} = require("../models/phenomenon.model");
+
+exports.getPhenomenon = async (req, res) => {
+    try{
+        res.status(200).send(req.Phenomenon);
```

Comment:
Maybe use `json` instead of `send` so that the content-type header is sent as 'application/json'

Reply:
From what I see I think she's following quite closely to the way Andy taught people. 
Sounds good though! I'm excited to see how it comes together.